packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -104,6 +104,9 @@ class GtagEventDevTools extends GtagEvent {
     int? heap_diff_objects_before, // metric7
     int? heap_diff_objects_after, // metric8
     int? heap_objects_total, // metric9
+    int? root_set_count, // metric10
+    int? row_count, // metric11
+    String? inspector_tree_controller_id, // metric12
```

Comment:
![Screenshot 2023-03-08 at 11 05 46 AM](https://user-images.githubusercontent.com/43759233/223813041-1a384206-a733-4b94-90ea-8dbb92429a04.png)

metrics can only be Integers, doubles, or time. We have two options here. We can make this an int: 0 for summary and 1 for details tree (this may be the easiest option)? Or we can make custom screen dimensions like we have made custom screen metrics, and make the inspector tree id a custom dimension